api_app/models/domain/costs.py

```diff
@@ -0,0 +1,187 @@
+from datetime import datetime, timedelta, date
+from typing import List, Optional
+from pydantic import BaseModel
+from enum import Enum
+import random
+import uuid
+
+
+class GranularityEnum(str, Enum):
+    daily = "Daily"
+    none = "None"
+
+
+def generate_cost_row_example(granularity: GranularityEnum):
+    return dict({
+        "cost": random.uniform(0, 1000), "currency": "USD", "date":
+            (datetime.today() - timedelta(
+                days=-1 * random.randint(0, 1000))).date() if granularity == GranularityEnum.daily else None
+    })
+
+
+def generate_cost_item_example(name: str, granularity: GranularityEnum):
+    cost_item = dict(
```

Comment:
unclear name because it doesn't return cost item - in all example methods

Reply:
all ```dict``` typed functions and locals renamed for clarity